src/libraries/CCTPLib.sol

```diff
@@ -91,6 +91,8 @@ library CCTPLib {
     /*** Relayer helper functions                                                               ***/
     /**********************************************************************************************/
 
+    // NOTE: As we are transferring USDC using CCTP, we are not using the `_forceApprove` logic
```

Comment:
Avoid first-person language in all comments (e.g., "we", "us" etc).

`// NOTE: As USDC is the only asset transferred using CCTP,  `_forceApprove` logic is not needed`

Reply:
fixed